datasets/streaming/error_aggregates/reference.md

```diff
@@ -0,0 +1,93 @@
+# Error Aggregates Reference
+
+<!-- toc -->
+
+# Introduction
+
+{% include "./intro.md" %}
+
+# Data Reference
+
+## Sampling
+
+### Invalid Pings
+
+We ignore invalid pings in our processing. Invalid pings are defined as those that:
+
+* The submission dates or activity dates are invalid or missing.
+* The build ID is malformed.
+* The docType field is missing or unknown.
+* The ping does not include a positive count of usage hours.
+
+## Scheduling
+
+The `error_aggregates` job is run continously, using the spark streaming infrastructure
```

Comment:
Is there a prod/docs link for our spark streaming infra?

Reply:
Not that I am aware of :/